src/librustc_resolve/resolve_imports.rs

```diff
@@ -683,9 +683,8 @@ impl<'a, 'b:'a> ImportResolver<'a, 'b> {
         };
 
         match (value_result, type_result) {
-            // With `#![feature(item_like_imports)]`, all namespaces
-            // must be re-exported with extra visibility for an error to occur.
-            (Ok(value_binding), Ok(type_binding)) if self.new_import_semantics => {
+            // All namespaces must be re-exported with extra visibility for an error to occur.
+            (Ok(value_binding), Ok(type_binding)) => {
```

Comment:
This expedites the stabilization of a small, backwards compatible part of RFC 1560 to fix the first issue from https://github.com/rust-lang/rust/issues/36768#issuecomment-249974646.